SoObjects/SOGo/SOGoSystemDefaults.m

```diff
@@ -163,6 +165,12 @@ + (void) prepareUserDefaults
   for (count = 0; count < sizeof(confFiles)/sizeof(confFiles[0]); count++)
     _injectConfigurationFromFile (configFromFiles, confFiles[count], logger);
 
+  /* Fill/Override configuration with configuration stored in the extra
+   *  config file */
+  envFile = [[[NSProcessInfo processInfo] environment] objectForKey:@"SOGO_EXTRA_CONFIG_FILE"];
```

Comment:
It would be helpful if this was documented

Reply:
I totally agree. Do you have any pointer where an appropriate location would be?

Reply:
I also agree the documentation should be updated. I suggest modifying this part : https://github.com/Alinto/sogo/blob/master/Documentation/SOGoInstallationGuide.asciidoc#gnustep-environment-overview